sftp-connector-demo/src/META-INF/keys/sftpConnector_rsa.ppk

```diff
@@ -0,0 +1,26 @@
+PuTTY-User-Key-File-2: ssh-rsa
+Encryption: aes256-cbc
+Comment: rsa-key-20210811
+Public-Lines: 6
+AAAAB3NzaC1yc2EAAAABJQAAAQEAt+DtLc7p5lx5erhsHUq/Zsx4A3qNpSwQmAhM

```

Comment:
I think we should not add any public/private keys to the demo. They belong to the user and must match the target system: so I don't see any value put a possible security leak in adding keys to the SCM.

Reply:
I put them for the Demo to accelerate testing purposes but I will delete them both.